logic/vote.js

```diff
@@ -140,6 +140,11 @@ Vote.prototype.calculateFee = function() {
  * @todo Add description for the params
  */
 Vote.prototype.verify = function(transaction, sender, cb, tx) {
+	const amount = new Bignum(transaction.amount);
+	if (amount.greaterThan(0)) {
```

Comment:
@nazarhussain In testnet we have a transaction that will fail this check, please make sure if it's in exceptions:
```
lisk_test=# select id, type, amount, "blockId" from trs where amount > 0 AND type > 0 AND type < 6;
          id          | type |  amount   |       blockId       
----------------------+------+-----------+---------------------
 15449731671927352923 |    3 | 100000000 | 2528992701257192568
(1 row)
```